zad17.py

```diff
@@ -1,51 +1,26 @@
-def zad17(a,b):
-
-    def is_prime(n):
-        if n < 2:
-            return False
-        elif n == 2:
-            return True
-        elif n % 2 == 0:
-            return False
-
-        div = 3
-        while div * div <= n:
-            if n % div == 0:
-                return False
-            div += 2
-        return True
+from is_prime import is_prime
 
-    def num_len(n):
```

Comment:
To `num_len` nie było używane co nie?

Reply:
W pierwszym pomyśle było, ale później po prostu zapomniałem go usunąć.